pkgs/cli/src/constants.zig

```diff
@@ -23,3 +23,6 @@ pub const DEFAULT_NODE_KEY: []const u8 = "./key";
 
 /// Maximum size (in bytes) for hash-sig key JSON blobs ingested by the CLI
 pub const MAX_HASH_SIG_KEY_JSON_SIZE: usize = 128 * 1024 * 1024;
+
+/// Maximum size (in bytes) for hash-sig key SSZ blobs ingested by the CLI
+pub const MAX_HASH_SIG_KEY_SSZ_SIZE: usize = 128 * 1024 * 1024;
```

Comment:
it's exactly the same as `MAX_HASH_SIG_KEY_JSON_SIZE` and it's huuuuuuuuuuuuge.

If it really makes sense to have the exact same value for a binary encoding as for the JSON encoding, then just rename the variable above `MAX_HASH_SIG_ENCODED_KEY_SIZE`. If it really makes sense to differentiate the two, use a smaller value.